homeassistant/components/insteon/__init__.py

```diff
@@ -51,7 +51,7 @@ async def async_get_device_config(hass, config_entry):
 
     load_aldb = 2 if devices.modem.aldb.read_write_mode == ReadWriteMode.UNKNOWN else 1
     await devices.async_load(id_devices=1, load_modem_aldb=load_aldb)
-    for addr in devices:
+    for addr in list(devices):
```

Comment:
I guess this is not related to added support for scenes?
Please either move unrelated changes to a separate PR or add review comments in this PR to make reviewing easier.

Reply:
I reverted the two unrelated changes. This one and one in ./api/__init__.py to remove unused code.